app/src/main/java/com/example/zingmp3phake/utils/Constant.kt

```diff
@@ -28,6 +28,8 @@ const val TAG_LOG = "AAAAAA"
 const val N0_LYRIC = "Chưa có lời cho bài hát này"
 const val NO_DATA = "No Data Founded"
 const val NO_INTERNET = "Không có kết nối mạng"
+const val PAGE_SIZE = 10
+const val UTF = "utf-8"
```

Comment:
```suggestion
const val UTF = "utf-8"

```

add 1 bank line phân cách giữa hằng số & biến thông thường cho nó dễ đọc